packages/twenty-server/src/engine/metadata-modules/object-metadata/services/object-metadata-migration.service.ts

```diff
@@ -118,163 +115,55 @@ export class ObjectMetadataMigrationService {
     );
   }
 
-  public async createStandardRelationsUpdatesMigrations(
+  public async createUpdateForeignKeysMigrations(
```

Comment:
(discussed offline but to keep a trace) 
I understand that it brings clarity to treat metadata updates and migrations separately, but I feel it makes the object-metadata-migration api a little less accessible perhaps, as actually from my understanding  updateRelationsAndForeignKeysMetadata should never be called without createUpdateForeignKeysMigrations too, or we will have a discrepancy between metadata and migration